odk2-src/survey-sample-app-install.rst

```diff
@@ -13,15 +13,15 @@ To access the sample application and its six sample forms:
       :alt: Survey Setting Button
       :class: device-screen-vertical
 
-  2. Follow the :ref:`services-managing-server-config` instructions to set up your server.
+  2. Select Settings -> Server Settings
```

Comment:
I like your improvement of adding the actual words. I would add back *in parenthesis* the  link to the picture so people who are confused have a visual.

I do agree following the link as the primary route is confusing, but leaving it as a secondary clarification might not be a bad thing